### ISSUE TITLE
Is there a way to support Microsoft.OData Attributes?

### ISSUE THREAD

**Issue by RichardMau5** [object/Simple.OData.Client#561]
Using automated code generation, every generated property looks like this:
```
[global::Microsoft.OData.Client.OriginalNameAttribute("name")]
        public string Name {get; set;}
...
```

However, this is not supported by Simple.OData, so I could manually replace it to `[DataMember(Name = "name")]` But this is cumbersome.
Is there a better way to do this? Can this be included, or should I write my own program to create classes from CSDL

**Comment by phatcher** [comment]
@RichardMau5 If you look at the code in MemberInfoExtensions you can see how we currently support  attrbutes so we can perform the mapping.

Shouldn't be too tough to extend this but please provide some tests that prove the result.

**Comment by RichardMau5** [comment]
Yeah this should be possible, definately will consider adding this. Just at `var propertyName = property.Name;` is where we will maybe have to do something different
